KarhooUISDK.xcodeproj/project.pbxproj

```diff
@@ -2075,6 +2080,7 @@
 				407EC73922CF564E00F24CFC /* Attributes.swift */,
 				407EC73822CF564E00F24CFC /* Constraints.swift */,
 				407EC73A22CF564E00F24CFC /* KarhooColors.swift */,
+				518245862790A917006B791A /* KarhooColors+defaultSet.swift */,
```

Comment:
Could you name this KarhooColors+**D**efaultSet? I.e. with an uppercase D

Reply:
Sure